apps/remix-ide/src/app/tabs/locales/en/panel.json

```diff
@@ -4,6 +4,8 @@
   "panel.documentation": "Documentation",
   "panel.description": "Description",
   "panel.maintainedByRemix": "Maintained by Remix",
+  "panel.pinnedMsg": "Click to dock plugin to the right",
+  "panel.unPinnedMsg": "Click to return plugin to the side panel",
```

Comment:
may be better to say: `Click to return plugin to the left side panel`

Reply:
Agree. Maybe even same sentence but with right/left